docs/user-authentication.txt

```diff
@@ -97,8 +97,64 @@ manage user authentication, as shown in the following code:
    :language: php
    :dedent:
 
-Enable Password Reminders 
--------------------------
+Customize User Authentication
+-----------------------------
+
+In this section, you can learn how to use the following features to
```

Comment:
S: before this sentence, add a sentence explaining why someone might want to make a custom model